README.md

```diff
@@ -11,6 +11,29 @@ in [CommCare HQ](http://github.com/dimagi/commcare-hq).
 Image courtesy of the [ReMIND
 project](https://www.commcarehq.org/exchange/325775003aa58cfcefbc75cfdf132e4d/info/).
 
+Vocabulary
+----------
+
+Some of the names used in the source code are less than intuitive. Hopefully
+this list will help to reduce the confusion. The list is ordered with the least
+intuitive items first.
+
+- **Vellum**: also known as the _Form Builder_.
+- **Mug**: an object representing a question. Each mug has a type: _Text_,
+  _Date_, _Audio_, etc. While some mug type names match the corresponding label
+  used in the UI, some do not. For example, a _Trigger_ is called a _Label_ in
+  the UI.
+- **JavaRosa**: the language/translation module. A core part of the JavaRosa
+  module is the **IText** system, which provides an API for translated strings
+  and multimedia used to adorn questions.
+- **Widget**: a control or group of controls displayed on the right side of the
```

Comment:
example?

Reply:
The XPath expression widget in the image below is outlined in red
![image](https://user-images.githubusercontent.com/464726/62649596-1c38f700-b923-11e9-97c4-d5176a214409.png)